medmodels/matching/metrics.py

```diff
@@ -63,6 +65,43 @@ def exact_metric(vector1: NDArray[np.float64], vector2: NDArray[np.float64]) ->
     return np.inf
 
 
+def mahalanobis_metric(*vectors: NDArray[np.float64], inv_cov: NDArray[np.float64]):
```

Comment:
oh and `*vectors` should probably be refactored into `vector_1` and `vector_2` or something similar